packages/terra-docs/src/terra-dev-site/tool/DocumentationComponents/Notice.tool.mdx

```diff
@@ -54,15 +55,19 @@ This component was created for this express purpose and caution should be used w
 
 <ImportantNoticeVariant title="Important Notice Variant" description="This notice type will be used most often. It should be used for any kind of tip, helpful note, suggestion, reminder and example for how to implement accessibility correctly, reminder to add a prop, etc."/>
 
-<CautionNoticeVariant title="Caution Notice Variant" description="This notice type should be used very rarely when we need to list a warning for when a prop (or combination or props) are used incorrectly and will cause instability, throw warnings, or outright break."/>
+<CautionNoticeVariant title="Caution Notice Variant" description="This notice type should be used when needing to list a warning for when a prop (or combination or props) if used incorrectly will cause instability, may throw warnings, and should be avoided."/>
 
 <UXRecommendationNoticeVariant title="UX Recommendation Notice Variant" description="This notice type should be used very rarely to point out something extremely important as a usage recommendation. It should not to be used on every page. The messaging should be UX or Human Factors focused, and not more than paragraph or two, sometimes with links to other industry resources."/>
 
 ## Notices for Development Support Status
 
 <MaintenanceNoticeVariant title="Maintenance Notice Variant" description="This notice type should be used for components that will be deprecated at some future time. This notice should be placed directly under the component title in documentation."/>
 
-<DeprecationNoticeVariant title="Deprecation Notice Variant"description="This notice type should be used for components that is deprecated and will receive no further updates. This notice should be placed directly under the component title in documentation." />
+<DeprecationNoticeVariant title="Deprecation Notice Variant" description="This notice type should be used for components that is deprecated and will receive no further updates. This notice should be placed directly under the component title in documentation." />
```

Comment:
Excellent - this will play into https://github.com/cerner/terra-ui/issues/413 quite nicely.

Reply:
That's what both the "Maintenance" and "Deprecation" notice types were created for -- so we can put then at the top of our documentation page with the same message that gets put in the banner on the NPM site for deprecated components. https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions

But your highlighting made me spot an old grammer mistake, so gave it a quick fix: 
[...] for components that ~~is~~ **are** deprecated and will [...] https://github.com/cerner/terra-core/pull/3584/commits/6bad5d85a088fa636fa973929ee1965e5eb593c5